snapcraft.yaml

```diff
@@ -86,7 +95,9 @@ parts:
     parse-info: [usr/share/metainfo/org.gnome.Epiphany.appdata.xml]
     meson-parameters:
       - --prefix=/usr
+    build-snaps: [core22, gtk-common-themes, gnome-42-2204]
     override-build: |
+      $CRAFT_PROJECT_DIR/snaptools/remove_common.py core22 gtk-common-themes gnome-42-2204
```

Comment:
the path should be `$CRAFT_PROJECT_DIR/snaptools/remove_common/remove_common.py`